back-end/resources/manuals/room_manual.md

```diff
@@ -145,7 +145,7 @@ This can be displayed in the front-end, so should be readable and in Dutch.
     - `message` in case of type `device`: this defines a list of componentInstructions which have a:
         - `component_id`: this will be the id of a component in a timer or device
         - `instruction`: one of the instructions specified for this device and component
-        - `value`: this is the value for the instruction of the type specified for this device and component
+        - `value`: this is the value for the instruction of the type specified for this device and component, in case this type is "status", a string in the form "`device_id`.`component_id`" should be specified. 
```

Comment:
probably more clear as `device_id.component_id`

Reply:
dat staat er toch?

Reply:
Current is `device`. `component`

Reply:
done